geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/string/AbstractAppendMemoryIntegrationTest.java

```diff
@@ -59,7 +60,8 @@ public void testAppend_actuallyIncreasesBucketSize() {
     info = RedisTestHelper.getInfo(jedis);
     Long finalMemValue = Long.valueOf(info.get("used_memory"));
 
-    assertThat(finalMemValue).isGreaterThan(previousMemValue);
+    GeodeAwaitility.await().atMost(Duration.ofSeconds(20))
+        .untilAsserted(() -> assertThat(finalMemValue).isGreaterThan(previousMemValue));
```

Comment:
This test is going to be flaky some of the time. If you happened to sample `used_memory` at a time just before we do a GC, it may take a long time or we may never sample a value of used memory that is less than that value.